### PR TITLE
Lab11

### PR DESCRIPTION
**Pull request by none-word**

### CODE REVIEW

File: k8s/helm-app-python/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+             - name: secret
```
Comment (AKurmazov) [inline k8s/helm-app-python/templates/deployment.yaml:37]: nit: Environmental variables are conventionally called in the UPPER_CASE

**Comment by AKurmazov** [comment]
Well done! 👍

**Comment by Maxkoz777** [comment]
Good job!

**Comment by afonyaa** [comment]
Great!

**Pull request closed without merging**